logging/logrus/options.go

```diff
@@ -95,6 +98,16 @@ func WithResponseBodyCapture(deciderFunc func(r *http.Request, status int) bool)
 	}
 }
 
+// WithDecider customizes the function for deciding if the middleware logs at the end of the request.
+func WithDecider(f Decider) Option {
+	return func(o *options) {
+		o.shouldLog = f
```

Comment:
how about if we went for something more specific and have a option for `PathToExclude`? I don't think we need a function this generic atm.

Reply:
We have a specific use case where we want to suppress logs where the status code is between `400` and `500`.